Packages/Text Integration Utility/Routines/TIULF.m

```diff
@@ -1,5 +1,8 @@
-TIULF ; SLC/JER - More computational functions ; 4/2/03
- ;;1.0;TEXT INTEGRATION UTILITIES;**162**;Jun 20, 1997
+TIULF ; SLC/JER - More computational functions ;9월 06, 2018@15:15
```

Comment:
any reason for a utf8 character here?

Reply:
I'm a consistency person, so i'd like for it not to be changed, but your choice :)